fastlane/lib/fastlane/helper/crashlytics_helper.rb

```diff
@@ -72,8 +72,12 @@ def download_android_tools
           begin
             UI.important("Downloading Crashlytics Support Library - this might take a minute...")
 
-            result = Net::HTTP.get(URI(url))
-            File.write(zip_path, result)
+            uri = URI(url)
+            http_conn = Net::HTTP.new(uri.host, uri.port)
+            http_conn.use_ssl = true
+            result = http_conn.request_get(uri.path)
+            raise "#{result.message} (#{result.code})" unless result.kind_of? Net::HTTPSuccess
```

Comment:
Please use `UI.user_error!` instead of `raise` as described here: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/UI.md